azure/pipelines/release.yml

```diff
@@ -97,6 +97,7 @@ stages:
       dsiApiSecret: '$(dsiApiSecret)'
       ucasUsername: '$(ucasUsername)'
       ucasPassword: '$(ucasPassword)'
+      ucasZipPassword: '$(ucasZipPassword)'
```

Comment:
Ditto, see comment on build.yml

Reply:
Same as https://github.com/DFE-Digital/apply-for-teacher-training/pull/2484#discussion_r457368555